components/character.py

```diff
@@ -0,0 +1,12 @@
+class Character:
+    def __init__(self, **kwargs):
+        self.id = id
+        self.name = name
+        # ?????? what's the difference?
+        self.active_mouth = False
```

Comment:
this wouldn't be a boolean, this would be an image. we can set this to None for now